compiler/control/OMROptions.cpp

```diff
@@ -4065,6 +4112,23 @@ static int32_t    count[numHotnessLevels] = {-2};
 static int32_t   bcount[numHotnessLevels] = {-2};
 static int32_t milcount[numHotnessLevels] = {-2};
 
+/**
+ * Merge option sets stored in _postRestoreOptionSets into _optionSets.
+ * _postRestoreOptionSets is then set to NULL.
+ */
+void
+OMR::Options::mergePostRestoreOptionSets()
+   {
+   TR::OptionSet *optionSet = _postRestoreOptionSets;
+   while (optionSet)
+      {
+      TR::OptionSet *next = optionSet->getNext();
+      self()->addOptionSet(optionSet);
+      optionSet = next;
+      }
```

Comment:
Because both `addOptionSet` and `addPostRestoreOptionSet` add to front, one can't just take the tail of `_postRestoreOptionSets` and attach it to the head of `_optionSets` without looping through all of `_postRestoreOptionSets` anyway, so I figured this was the cleanest approach.